packages/gatsby-source-wordpress/README.md

```diff
@@ -86,6 +86,14 @@ The Authentication on Wordpress.com is not supported yet. This means that you wo
           user: 'your-htaccess-username',
           pass: 'your-htaccess-password',
           sendImmediately: false
+
+          // If hostingWPCOM is true then you will need 
+          // to communicate with wordpress.com api
+          // in order to do that you need to create an app
+          // at https://developer.wordpress.com/apps/
+          // then add the clientId and clientSecret here
+          clientId: '',
```

Comment:
I had it namespaced when i started,  but I didn't find any more uses for the oAuth so I just kept it in the auth where it made more sense.

But I will happily do it.  What should I name the namespace?